README.md

```diff
@@ -1,6 +1,6 @@
 # This package contains the Enclosure LED Utilities, version 0.97
```

Comment:
This package contains the Enclosure LED Utilities, version 1.0.0

Reply:
Done